libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1357,8 +1357,7 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 			&dynamic_cast<ContractType const&>(*functionType->returnParameterTypes().front()).contractDefinition();
 		m_context.subObjectsCreated().insert(contract);
 
-		Whiskers t(R"(
-			let <memPos> := <allocateUnbounded>()
+		Whiskers t(R"(let <memPos> := <allocateUnbounded>()
```

Comment:
This new line was removed to avoid that we have a blank line between the source location comment and the code